features/global/layout.erb

```diff
@@ -4,6 +4,7 @@
     <title>fastlane CI</title>
     <link rel="stylesheet" href="/material.min.css">
     <script src="/material.min.js"></script>
+    <script src="/jquery-3.3.1.min.js"></script>
```

Comment:
I've added `jQuery` for ease `DOM` manipulation, but this is not a change that affects the application architecture so I thought it was safe to add for development purposes.

Reply:
Sorry to be that person, but I'm pretty sure we want to avoid jQuery :)

Reply:
Do you have another suggestion during early development stages besides raw JS for `DOM` manipulation? I'm totally open for other options, or just Vanilla JS if no other option is available

Reply:
I think because the only features we need are to query a json-serving endpoint and append items to the DOM, keeping it to vanilla js is the best way to move forward.  Eg: 

```js
<script type="text/javascript">
function loadXMLDoc() {
    var xmlhttp = new XMLHttpRequest();

    xmlhttp.onreadystatechange = function() {
        if (xmlhttp.readyState == XMLHttpRequest.DONE) {   // XMLHttpRequest.DONE == 4
           if (xmlhttp.status == 200) {
               document.getElementById("myDiv").innerHTML = xmlhttp.responseText;
           }
           else if (xmlhttp.status == 400) {
              alert('There was an error 400');
           }
           else {
               alert('something else other than 200 was returned');
           }
        }
    };

    xmlhttp.open("GET", "ajax_info.txt", true);
    xmlhttp.send();
}
</script>
```
(taken from https://stackoverflow.com/a/8567149)

Reply:
Alright, gonna stick with vanilla for now, was using jQuery to avoid wasting time on things that doesn't really matter, as I have tested that the service itself works (I've really never worked with vanilla before😅)

Reply:
Awesome, thanks @minuscorp, I know it's a pain, but keeping as many external libraries out as possible helps us reduce the chances of running into support issues in the future. Also, you get to learn something 😄 (even if you might not ever use it again 🙃)

Reply:
I couldn't remember why I hated vanilla JS until now 🤣